File: 2019paperhasher.js
let site = "www.ocr.org.uk";

let hash = (str) => {
    str = str.toUpperCase();
    str = str.split(".")[1];
    str = str.split(".")[0];
    //loop through string and add up the ascii values
    let total = 0;
    for (let i = 0; i < str.length; i++) {
        total += str.charCodeAt(i);
    }
    return total;
}

console.log(hash(site));

File: 2019paperhasher.py
sitename = "www.orc.org"


def hasher(site):
    site = site.split(".")[1]
    site = site.split(".")[0]
    site = site.upper()
    site = [ord(c) for c in site]
    total = 0
    for i in site:
        total += i
    return total


print(hasher(sitename))
